### PR TITLE
feat: Update docs reflecting `Fluence JS` -> `JS Client` update

### PR DESCRIPTION
**Pull request by coder11**

### CODE REVIEW

File: docs/build/js-client/4-call-functions.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- write aqua
+- export function
+- compile
+- call from code
+
+## Understanding the compiler output
+
+For every exported function definition in aqua the compiler generated two overloads. One accepting the `FluencePeer` instance as the first argument, and one without it. Otherwise arguments are the same and correspond to the arguments of aqua functions. The last argument is always an optional config object with the following properties:
+
+- `ttl`: Optional parameter which specify TTL (time to live) of particle with execution logic for the function
```
Comment (mikhail-1e20) [inline docs/build/js-client/4-call-functions.md]: 1. Why a list is introduced if it has the only member (`ttl`)?
 2. The previous sentence ends with "the following properties:" but just an optional parameter is described. How comes?

File: docs/build/js-client/5-register-services.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+// (4)
+export function registerServiceName(
+  peer: FluencePeer,
+  serviceId: string,
+  service: ServiceNameDef
+): void;
+```
+
+1. Uses default Fluence Peer and the default id taken from aqua definition
```
Comment (mikhail-1e20) [inline docs/build/js-client/5-register-services.md]: I'm not sure if  "aqua definition" needs an article or not.

File: docs/build/js-client/5-register-services.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+1. Uses default Fluence Peer and specifies the service id explicitly
+2. Specifying both peer and the service id.
+
+### Type conversion
+
+Same as in call function. But no arrow types
+
+### Service interface
+
+The service interface type follows closely the definition in aqua code. It has the form of the object which keys correspond to the names of service members and the values are functions of the type translated from aqua definition (see [Type conversion](#type-conversion)). For example, for the following aqua definition:
```
Comment (mikhail-1e20) [inline docs/build/js-client/5-register-services.md]: It is unclear whether "For example..." and "The typescript interface..." below make one sentence or not. If not, I would suggest to state explicitly what happens in "For example..."

File: docs/build/js-client/6-call-params.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Security andd call params
+
+# Call params, tetraplets and security
+
+Each service call is accompanied by additional information specific to Fluence Protocol. Including `initPeerId` - the peer which initiated the particle execution, particle signature and most importantly security tetraplets. All this data is contained inside the last `callParams` argument in every generated function definition. These data is passed to the handler on each function call can be used in the application.
```
Comment (mikhail-1e20) [inline docs/build/js-client/6-call-params.md]: The last sentence is unclear. It seems a word is lost: "This data is passed to the handler on each function call ??? can be used in the application."

File: docs/build/js-client/4-call-functions.md (hunk 1 above)
Comment (coder11) [reply docs/build/js-client/4-call-functions.md]: Thanks for you comment. I think we have to postpone these section to after the Devner event

File: docs/build/js-client/6-call-params.md (hunk 4 above)
Comment (coder11) [reply docs/build/js-client/6-call-params.md]: Thanks for you comment. I think we have to postpone these section to after the Devner event

File: docs/build/js-client/5-register-services.md (hunk 3 above)
Comment (coder11) [reply docs/build/js-client/5-register-services.md]: Thanks for you comment. I think we have to postpone these section to after the Devner event

**Review by meph — APPROVED** [review]

**Pull request merged**